src/main/java/org/commcare/session/RemoteQuerySessionManager.java

```diff
@@ -124,7 +124,9 @@ public URL getBaseUrl() {
             for (Enumeration e = userAnswers.keys(); e.hasMoreElements(); ) {
                 String key = (String)e.nextElement();
                 String value = userAnswers.get(key);
-                if (!StringUtils.isEmpty(value)) {
+                QueryPrompt prompt = queryDatum.getUserQueryPrompts().get(key);
+                boolean exclude = prompt.getExclude();
+                if (!StringUtils.isEmpty(value) && !exclude) {
```

Comment:
evaluate the xpath to determine what the value will be
```suggestion
                XPathExpression excludeExpr = prompt.getExclude();
                if (excludeExpr == null || !(boolean) excludeExpr.eval(evaluationContext)) {
```